verifier/src/errors.rs

```diff
@@ -74,6 +77,13 @@ impl fmt::Display for VerifierError {
             Self::FriVerificationFailed(err) => {
                 write!(f, "verification of low-degree proof failed: {err}")
             }
+            Self::InsufficientConjecturedSecurity(minimal_security,proofs_security)=> {
+                write!(f, "verification failed due to parameters not providing enough conjectured security: expected at least {minimal_security} bits of security but the proof provides only {proofs_security} bits of conjectured security")
+            }
```

Comment:
nit: I would probable be a bit more concise than this. For example: "insufficient proof security level: expected at least {minimal_security} bits of conjectured security, but was {proof_security} bits"

Also, I would add a space after the comma in "minimal_security,proofs_security" on line 80.